health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/hooks/services/createUpdatePlanProject.js

```
@@ -716,26 +748,26 @@ const createUpdatePlanProject = async (req) => {
-        });
+        const formulasToUpdateFromUIForSubFormula = req?.formulasToUpdate
+          ?.filter((row) => {
+            return row.category && row.output && row.input && row.operatorName && row.assumptionValue;
+          })
+          ?.map((row) => {
+            const updatedRow = { ...row };
+            const operatorName = row?.operatorName;
+            delete updatedRow?.operatorName;
+            updatedRow.operator = state?.RuleConfigureOperators?.find((operation) => operation.operatorName === operatorName)?.operatorCode;
+            return updatedRow;
+          });
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)